lib/appmap/command/agent_setup/config.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'yaml'
+require 'appmap/service/guesser'
+
+module AppMap
+  module Command
+    module AgentSetup
+      ConfigStruct = Struct.new(:config_file)
+
+      class Config < ConfigStruct
+        def perform
+          if File.exist?(config_file)
+            puts "AppMap configuration file already exists at #{config_file}"
+            exit(1)
```

Comment:
I feel like process exit should be handled in the `exe` package? So maybe just raise or return false.

Reply:
I've changed it to raise an error